README.md

```diff
@@ -17,42 +17,29 @@ Reference:
 
 1. Configure the project to use the Edge Runtime:
```

Comment:
You've removed the runtime config in the next.config.js and just exporting the config in the hello.ts file, doesn't that make the config apply only when you try to use the hello endpoint? (and the rest of the app still uses the default node runtime)

Reply:
I think this is fine as the vp of dx at Vercel suggested that setting the runtime globally is not recommened (https://github.com/cloudflare/cloudflare-docs/pull/7097#issue-1508782427)

Reply:
Hey @dario-piotrowicz thanks for the link. Nice catch.